tests/process.rs

```diff
@@ -19,3 +19,25 @@ fn test_process() {
         .values()
         .any(|p| p.exe().to_str().unwrap_or_else(|| "").len() != 0));
 }
+
+#[test]
+fn test_process_disk_usage(){
+    use sysinfo::{ProcessExt, SystemExt};
+    use std::fs::File;
+    use std::fs;
+    use std::io::prelude::*;
+    {
+        let mut file = File::create("test.txt").unwrap();
+        file.write_all(b"This is a test file\nwith test data.\n").unwrap();
+    }
+    fs::remove_file("test.txt").ok();
+    let mut system = sysinfo::System::new();
+    system.refresh_processes();
+    let process_list = system.get_process_list();
+    let mut write_bytes: u64 = 0;
+    for p in process_list.values(){
```

Comment:
In here, if any process wrote any data, it'll be > 0. I assume this is not what you want to test. So instead, just get the current pid and do:

```rust
let p = system.get_process(sysinfo::get_current_pid().expect("failed to get current pid")).expect("no such process");
assert!(p.written_bytes() > 0);
```